rules/S6232/cfamily/metadata.json

```
@@ -1,13 +1,12 @@
 {
-  "title": "\"std::bit_cast\" should be used instead of union type-punning",
+  "title": "Well defined type-punning should be used instead of union type-punning",
```

"type-punning" is quite a heavy word to repeat in a single sentence.
What do you think of the following options?
```suggestion
  "title": "Well-defined alternative should be used instead of union type-punning",
```
```suggestion
  "title": "Well-defined type-punning method should be used instead of a union-based one",
```